blog/tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""This package contains tests for project."""
```

Comment:
Don't ever refer to self via "This ...". It is obvious that if the docstring refers to the scope where it resides.
Writing explanations this way is useless.